test/services/helpers/scopes/hooks/checkScopePermissions.js

```diff
@@ -0,0 +1,180 @@
+const { expect } = require('chai');
+const { BadRequest, Forbidden } = require('@feathersjs/errors');
+const { ObjectId } = require('mongoose').Types;
+
+const {
+	getScopePermissions,
+	checkScopePermissions,
+} = require('../../../../../src/services/helpers/scopePermissions/hooks/checkScopePermissions');
+
+const ALLOWED_USER_ID = (new ObjectId()).toString();
+const ALLOWED_USER_PERMISSIONS = ['EAT', 'THROW_AWAY'];
+const EXISTING_SCOPE = 'chocolateBars';
+const getApp = (omitHandler) => ({
+	service: (name) => {
```

Comment:
Is it a part for the testObjects. To fake a app with one services?

Reply:
Maybe. I'll think about it